public/assets/main.css

```diff
@@ -209,3 +196,38 @@ input#chatbar {
   dominant-baseline: middle;
   pointer-events: none;
 }
+
+.title {
+  font-weight: bold;
+  text-align: center;
+  padding: 0.3rem;
+}
+
+.card {
+  display: inline-block;
+  overflow: hidden;
+  vertical-align: top;
+  width: 24rem;
+  margin: 0.5rem 0.5rem 0 0;
+  border: 1px solid gainsboro;
+  border-radius: 0.7rem;
+}
+.card table {
+  line-height: 1.25rem;
+}
+.card th {
+  padding: 0 0.3rem;
+}
+.card td {
+  padding: 0 0.5rem;
+}
+
+.corp div {
+  display: grid;
+  align-self: center;
+  justify-self: center;
+}
+table.shareholders, table.companies, table.revenue {
```

Comment:
should we have these?

Reply:
it saves three props

Reply:
not worth it :D

Reply:
renamed class to center, moved collapse to prop